src/Symfony/Component/Config/Definition/Builder/ExprBuilder.php

```diff
@@ -98,6 +98,18 @@ public function ifNull()
     }
 
     /**
+     * Tests if the value is empty.
+     *
+     * @return ExprBuilder
+     */
+    public function ifEmpty()
+    {
+        $this->ifPart = function ($v) { return empty($v); };
```

Comment:
As @stof said, it isn't an issue to me, because anyone using it must understand it behaves like its php `empty` function counterpart, and know the limitations.
If not, we have to rename it to `isEmptyArray`, or whatever, but right now I think the expression builder should stick with simple expressions like this.

If you really expect `''`, `false`, `null`, `'0'` or `0` to be valid for your node, you shouldn't use `ifEmpty`.